boards/arm/stm32mp157c_dk2/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Kconfig - STM32MP157C Discovery board configuration
+#
+# Copyright (c) 2019 ST Microelectronics Limited
```

Comment:
Please change copyright to "STMircoelectronics".
"Limited" does not apply.
Apply everywhere